### PR TITLE
:seedling: add unit tests for events.go

### PR DESCRIPTION
**Pull request by khareyash05**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Adds tests for `pkg/synchromanager/clustersynchro/queue/event.go`

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/synchromanager/clustersynchro/queue/event_test.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+				reputCount: 1,
+				Action:     Added,
+				Object:     "Object1",
+			},
+			newer: &Event{
+				reputCount: 2,
+				Action:     Updated,
+				Object:     "Object2",
+			},
+			reputCount: 2,
```
Comment (Iceber) [inline pkg/synchromanager/clustersynchro/queue/event_test.go:28]: We might be better off with the full event as expected, even though the data and reputCount are the same as the newer, but the Action might change, for example here the event Action would be `Added`

Comment (khareyash05) [reply pkg/synchromanager/clustersynchro/queue/event_test.go:28]: I am not sure if I understood it correctly.Do I have to add more tests as well?

Comment (Iceber) [reply pkg/synchromanager/clustersynchro/queue/event_test.go:28]: I think we can use an expected event object to compare, e.g.
```go
			name: "Newer event is not nil",
			older: &Event{
				reputCount: 1,
				Action:     Added,
				Object:     "Object1",
			},
			newer: &Event{
				reputCount: 2,
				Action:     Updated,
				Object:     "Object2",
			},
			expected: Event{
				reputCount: 2,
				Action:     Added,
				Object:     "Object2",
			},
````

**Review by Iceber — APPROVED** [review]
LGTM, Thank you very much for contributing.

**Comment by Iceber** [comment]
@khareyash05 golang-lint is falied.

**Comment by khareyash05** [comment]
Fixed Lint.

**Pull request merged**